urllib3/util/ssl_.py

```diff
@@ -120,16 +120,14 @@ def assert_fingerprint(cert, fingerprint):
     """
 
     fingerprint = fingerprint.replace(':', '').lower()
-    digest_length, odd = divmod(len(fingerprint), 2)
```

Comment:
@sigmavirus24 Can you double check that we didn't ruin something important somehow?